### DISCUSSION TITLE
Windows 10 Start Menu - Tiles with a peeking image aren't "live"

### DISCUSSION THREAD

**Discussion by surya-sk** [valinet/ExplorerPatcher#1039] [General]
Live tiles with a peeking image and text aren't really live as the tile is just stuck on the first images. Examples are Groove Music, News and GroupMe. Apps where there is an image tile followed by some text. This would make the start menu perfect, because other than this, it is working beautifully. 
I just wanted to bring this to your attention. Great job on the start menu overall!

**Comment by valinet** [comment]
I mean, yeah, it's interesting they work even that much. 

To be honest, it's rather hard for me to understand how is it to code an overly complicated framework that breaks the first week after it's not a default in the OS anymore, while legacy stuff works just fine for 30+ years...

Live tiles are kind of broken, indeed. Idk, some work, some don't. I only ever had Weather pinned and it seems to still work and pull in data. For how long, idk. That's partly the reason I made the widget. Also, I use this menu because it is more logically thought out, with a list and an area to pin common shortcuts/things on the same page, rather than that backwards designed "improvement" this OS ships with by default. Needless to say, internals of how live tiles work were never documented, so neither do I know what could be causing this problem with them exactly, nor where to start looking, but honestly, yeah, I don't care, I am not looking to research this area more, it doesn't interest me. If someone else wants to do work in this area and maybe integrate it with EP, as always, I am more than happy to collaborate and maybe help upstream the patches.

Thanks

**Comment by surya-sk** [comment]
That's fair. I'll try and see if I can help in some way though I don't have much experience with this kind of stuff. There's this app on the Store called Live Tiles Anywhere that somehow replicated the tiles perfectly, but you can only pin them to the desktop rather than the Start menu. Does the job for now but maybe something worth looking into if you ever feel like it.

**Reply by valinet** [reply]
Yeah, I have briefly checked it out. It’s interesting. I wonder whether he reversed the actual libraries in Windows or just reimplemented very good looking lookalikes. If the former, considering the knowledge needed, then maybe this guy that worked on that app can take a look on the Windows 10 Start menu from here as well and make the tiles work reliably. We could collaborate. It’s difficult for me to know each and every thing, at least as it does take tons of time to research all these things. I an actually surprised and kind of sad that this project hasn’t attracted a developer audience as well…

**Comment by bbmaster123** [comment]
hey, I took I _very_ quick look and noticed if you full on replace appresolver.dll and starttiledata.dll with the older ones in system32, live tiles seem to work normally, and also pinning works from desktop, search, and file explorer. (only tested on latest dev build in my vm)
I only tested very briefly, but News switched between a few images, and weather allowed me to pin directly from within the app. These pins disappeared after restarting explorer, but they worked up until then, and the weather app still showed those locations as pinned. The original weather pin stayed and continued to work.

Maybe there are still a few particular COM interfaces or something still not getting redirected to the dlls in the systemapps folder? If globally replacing the files fixes the issue, that would seem like it would make sense, I think. 

I swear I will have more free time soon... its been a shit show on my end, and starting tomorrow its crunch time, but after that things should normalize for me again (hopefully) and I can be of maybe slightly more help.
